dataland-frontend/tests/e2e/utils/EuTaxonomyNonFinancialsUpload.ts

```diff
@@ -13,23 +13,68 @@ import Chainable = Cypress.Chainable;
  * Uploads a single eutaxonomy-non-financials data entry for a company via the Dataland upload form
  *
  * @param companyId The Id of the company to upload the dataset for
+ * @param valueFieldNotFilled Value which, if true, disables the value field
  * @returns the id of the dataset that has been uploaded
  */
-export function uploadEuTaxonomyDataForNonFinancialsViaForm(companyId: string): Cypress.Chainable<string> {
+export function uploadEuTaxonomyDataForNonFinancialsViaForm(
+  companyId: string,
+  valueFieldNotFilled = false
+): Cypress.Chainable<string> {
   cy.visitAndCheckAppMount(`/companies/${companyId}/frameworks/${DataTypeEnum.EutaxonomyNonFinancials}/upload`);
-  cy.get(`input[name="reportingPeriod"]`).type("2023");
-  cy.get("select[name=assurance]").select("Limited Assurance");
-  cy.get('input[id="reportingObligation-option-yes"][value=Yes]').check({
-    force: true,
+
+  cy.get('button[data-test="upload-files-button"]').click();
+  cy.get("input[type=file]").selectFile("tests/e2e/fixtures/pdfTest.pdf", { force: true });
+  cy.get('div[data-test="uploaded-files"]').find('[data-test="uploaded-files-title"]').should("contain", "pdf");
```

Comment:
Same comment as for financials:  You already got the file name in your path so you could also check for it here.

Reply:
ok

Reply:
Note for myself:   Put "pdfTest" into a const at the beginning

Reply:
Note: Do some refactoring on the "fill" and "upload" methods, as well as the "submitButton" and "uploadReports" const classes.